config/environments/local/local.node.config.toml

```diff
@@ -95,16 +95,14 @@ WeightArithmetics = 1
 WeightBinaries = 1
 WeightSteps = 1
 MaxAllowedFailedCounter = 50
-	[Sequencer.ProfitabilityChecker]
-		SendBatchesEvenWhenNotProfitable = "true"
 	[Sequencer.Finalizer]
-		GERDeadlineTimeoutInSec = 60
-		ForcedBatchDeadlineTimeoutInSec = 60
-		SendingToL1DeadlineTimeoutInSec = 60
-		SleepDurationInMs = 100
+		GERDeadlineTimeoutInSec = "60s"
+		ForcedBatchDeadlineTimeoutInSec = "60s"
+		SendingToL1DeadlineTimeoutInSec = "60s"
+		SleepDurationInMs = "100ms"
```

Comment:
I think you also need to include this config params into public.node.config.toml

Reply:
But I don't see anything for the Sequencer there. This is why I didn't add it there. Do we need a Sequencer config to be added in the public config?

Reply:
No, it is ok. I see that the sequencer component is missing in the public docker-compose too